cmd/manager/main.go

```diff
@@ -49,6 +50,7 @@ var (
 
 const (
 	defaultTimeout = time.Minute * 3
+	configFile     = "/run/config/csi-addons-config.json"
```

Comment:
why use a file, and not the Kubernetes API to access the ConfigMap directly? Users can then place keys directly in the ConfigMap, and do not need to worry about weird indirect json syntax.